src/view/pane/room.js

```diff
@@ -145,8 +141,7 @@ Candy.View.Pane = (function(self, $) {
            *   (String) roomJid - Room JID
            *   (jQuery.Element) element - Room element
            */
-          $(Candy).triggerHandler('candy:view.room.after-show', evtData);
-
+          $(Candy).triggerHandler('candy:view.room.after-show', roomObject);
```

Comment:
Convention suggests that the data param is a hash, so `{room: roomObject}`.

This also requires documentation updates.